infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/models/session/ApiSessionScope.kt

```diff
@@ -0,0 +1,48 @@
+package com.simprints.infra.eventsync.event.remote.models.session
+
+import androidx.annotation.Keep
+import com.simprints.infra.events.event.domain.models.Event
+import com.simprints.infra.events.event.domain.models.session.SessionScope
+import com.simprints.infra.eventsync.event.remote.models.ApiEvent
+import com.simprints.infra.eventsync.event.remote.models.ApiModality
+import com.simprints.infra.eventsync.event.remote.models.fromDomainToApi
+
+@Keep
+internal data class ApiSessionScope(
+    val id: String,
+    val projectId: String,
+    val startTime: Long,
```

Comment:
You are probably planning on doing this in a separate pull request but just in case: startTime and endTime should have the new timestamp format

Reply:
Yes, it is a whole different can of worms to open at a later date.